src/assets/js/partials/wishlist-card.js

```diff
@@ -0,0 +1,54 @@
+
+class WishlistCard extends HTMLElement {
+    constructor() {
+        super();
+        this.product = this.getAttribute('product')
+
+    }
+
+    connectedCallback() {
+        salla.onReady(() => {
+            this.render()
+        })
+    }
```

Comment:
- No need for `constructor`, `this.product` is already injected from the twilight salla-products-list,.
- Enhance it like this:
```
if(!this.product){
return salla.logger.warn('custom-wishlist-card:: product not existed!');
}
salla.onReady(() =>  this.render())
```